### ISSUE TITLE
Pytorch SAC alpha sign

### ISSUE THREAD

**Issue by jose-alatorre-harvard** [openai/spinningup#304]
Hi, 

In line 241  of  [sac.py ](https://github.com/openai/spinningup/blame/038665d62d569055401d91856abb287263096178/spinup/algos/pytorch/sac/sac.py#L215) 
`
        loss_pi = (alpha * logp_pi - q_pi).mean()
`

shouldnt be ?
`
      loss_pi = (q_pi-alpha * logp_pi ).mean()
`

I believe the signs are wrong.

**Comment by Harimus** [comment]
optimizing (taking learning steps) in pytorch are done using gradient **decent** on the loss function,  while in policy gradients methods,  we use gradient **ascent** to improve the actor. Switching the sign here is to make the gradient decent have the gradients ascent effect.  The `compute_loss(obs, act, weight)` in [Spinningup Intro3](https://spinningup.openai.com/en/latest/spinningup/rl_intro3.html#implementing-the-simplest-policy-gradient) uses the same trick.
